### ISSUE TITLE
Different game modes potentially not having intended behaviour(?)

### ISSUE THREAD

**Issue by agent30632** [Awedtan/LaserTag#49]
**Clarification**
When the game is set to game.mode != 0 (called **DM** in the code for now), enemies don't respawn at all. The thing is, when in this mode, the player doesn't permadeath, which kinda defeats the point(?). IMO, either the enemies should be respawning, or the player shouldn't be respawning. Furthermore, in game.mode = 0 (called **HSCORE** for the time being), the player permadies for the round, with the player being given a final score. Is this the intent of the gamemode, or do we want the player to respawn?

**TL;DR:** the game modes seem to have the opposite of the intended behaviour, unless this was intended, which is entirely another thing

**Comment by Awedtan** [comment]
i just forgot to make enemies respawn when mode isn't 0

**Comment by agent30632** [comment]
_b r u h_

**Comment by Awedtan** [comment]
fixed by dc6f0ee
